### DISCUSSION TITLE
nice

### DISCUSSION THREAD

**Discussion by HydenLiu** [hsl947/react-antd-multi-tabs-admin#7] [General]
刚开始学react，找了那么多模板学习，都太多东西太累赘，像ant-design-pro还要看umi之类的，暂时搞不动，其他的模板有很乱，结构一点都不清晰（可能是我太low了），不过这个挺好的，嘿嘿，可以愉快的搞起来了

**Comment by hsl947** [comment]
> 刚开始学react，找了那么多模板学习，都太多东西太累赘，像ant-design-pro还要看umi之类的，暂时搞不动，其他的模板有很乱，结构一点都不清晰（可能是我太low了），不过这个挺好的，嘿嘿，可以愉快的搞起来了

多谢支持哈，有什么好想法可以提~😁

**Comment by tearf001** [comment]
开通 赞助吧~~ 这个很精简了, 切换成最新的技术栈 react toolkit也容易

**Reply by hsl947** [reply]
> 开通 赞助吧~~ 这个很精简了, 切换成最新的技术栈 react toolkit也容易

中国大陆目前还不支持赞助，哈哈。
react toolkit有什么新特性吗，请教一下😁

**Reply by tearf001** [reply]
react toolkit 就是redux的 集成版本.  去掉一些冗长模板代码, 增强了一些封装. 内部就是个redux +react 其他社区库.

自称 "最佳实践"

**Reply by tearf001** [reply]
redux写 type和 actions真让人头大.我记忆力差. 
react toolkit 在redux上对 store/reducer 的分片, 这点很赞,当然不止这些...

你弄个二维码上去就可以收钱了.和街边卖红薯的差不多...

**Comment by tearf001** [comment]
oidc功能需不需要纳入特性? 对于OAuth验证提供集成还是个蛮新的点. 如果不用,也仍然可以用传统方式验证

**Reply by hsl947** [reply]
> oidc功能需不需要纳入特性? 对于OAuth验证提供集成还是个蛮新的点. 如果不用,也仍然可以用传统方式验证

oidc的可行性是没问题，再调研一下场景必要性，如果想要进一步提高安全性，可以把oidc纳入新特性。

**Comment by hsl947** [comment]
@tearf001 目前合并后，整体效果越来越赞了！
暂时发现了一些小优化，给你说一下😉

![1648911486(1)](https://user-images.githubusercontent.com/18359814/161389053-aff91b74-8915-43a1-983b-f4793d9da45d.png)
1、切换路由时，logo会闪动，导致整个导航栏移动后再还原；
2、这边的logo大小是不是有些不协调呢？😂
----
![image](https://user-images.githubusercontent.com/18359814/161389168-f4b18c1e-8b49-42ba-8534-519fe3ad2daf.png)
1、图标显示有些问题；
2、菜单看不见字了。
----
![image](https://user-images.githubusercontent.com/18359814/161389283-e70733a2-cc5e-44f6-81e7-2753dfda3903.png)
1、菜单折叠状态后，切换到横向导航栏模式，样式显示也异常了；😂
----
![image](https://user-images.githubusercontent.com/18359814/161389334-f0010ae0-24fb-4031-ae05-c1220cadf9c5.png)
1、标题也能选中了？
----

非常谢谢你百忙之中抽空帮忙维护并改进项目，当然我也希望不影响到你的正常工作。
有时间可以修改一下我上述问题，共同维护，一起进步吧。感谢。🤝
